types/office-js-preview/index.d.ts

```diff
@@ -77519,6 +77143,9 @@ declare namespace Word {
          * [Api set: WordApi 1.1]
          */
         interface ContentControlLoadOptions {
+            /**
+              Specifying `$all` for the LoadOptions loads all the scalar properties (e.g.: `Range.address`) but not the navigational properties (e.g.: `Range.format.fill.color`).
```

Comment:
I recommend that we not use `Range.address` as an example for interfaces other than Range. Either have an example that's relevant to the interface in question, or just not have a parenthetical example at all.

Reply:
Since this is already like this is Excel, let's have a separate conversation about addressing this string.

Reply:
ok